apify-docs-theme/src/theme/custom.css

```diff
@@ -1562,8 +1562,29 @@ html[data-theme='dark'] .runnable-code-block svg .apify-logo {
     text-align: right;
     min-width: 1.5rem;
     display: inline-block;
-    opacity: .3;
     left: var(--ifm-pre-padding);
+    color: var(--color-neutral-text-subtle);
+}
+
+.theme-code-block {
+    border-radius: 12px !important;
+    overflow: hidden;
+    border: 1px solid var(--color-neutral-border);
+    box-shadow: none !important;
+}
+
+.codeBlockTitle_node_modules-\@docusaurus-theme-classic-lib-theme-CodeBlock-Content-styles-module {
```

Comment:
i wonder if this actually works on a production build, probably not

Reply:
Good catch. It's hard to select the classes but this should do the trick.